### PR TITLE
Change xauth to xauthority

### PR DESCRIPTION
**Pull request by buschbapti**
I had the same problem than @domire8  so changing `XAUTH` to `XAUTHORITY` solved it. Otherwise docker had a problem mounting the volume. I also removed the `nvidia` dependency here that is not needed and changed the network to `host`.

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -1,25 +1,21 @@
...
-# change to false if not using nvidia graphic cards
-USE_NVIDIA_TOOLKIT=true
-
 NAME="aica-technology/ros2-intel-realsense"
 TAG="latest"
 
-[[ ${USE_NVIDIA_TOOLKIT} = true ]] && GPU_FLAG="--gpus all" || GPU_FLAG=""
+NETWORK="host"
 
 # By using --device-cgroup-rule flag we grant the docker continer permissions -
```
Comment (domire8) [inline run.sh:8]:
```suggestion
# By using --device-cgroup-rule flag we grant the docker container permissions -
```

File: run.sh [hunk 2]
```diff
@@ -1,25 +1,21 @@
...
 # to the camera and usb endpoints of the machine.
 # It also mounts the /dev directory of the host platform on the contianer 
 xhost +
 docker run -it --rm \
-    ${GPU_FLAG} \
+    --network ${NETWORK} \
     --privileged \
     -v /dev:/dev \
     --device-cgroup-rule "c 81:* rmw" \
```
Comment (domire8) [inline run.sh:16]: Is that really necessary? I didnt have to do that to get the PS3 controller input inside the container

**Review by domire8 — APPROVED** [review]

File: run.sh (hunk 2 above)
Comment (buschbapti) [reply run.sh:16]: this is from the official docker image intel is building so I assume it is

**Comment by buschbapti** [comment]
I am not sure if the volume mounting is also needed but in doubt I put it back

**Comment by buschbapti** [comment]
oh the other commit was supposed to be in a different branch. I put it there accidentally. But it corrects another typo anyway.

**Comment by domire8** [comment]
> I am not sure if the volume mounting is also needed but in doubt I put it back

I don't think it is, but I'm keeping it everywhere as well, just to be sure :smile:

**Review by eeberhard — APPROVED** [review]
Seems good to me! Since working with the realsense hardware should be done on only a linux machine, there doesn't need to be any OS specific flexibility in this run script. If it works for you then that's fine!

**Comment by buschbapti** [comment]
It does. And this change need to be applied in every image that is using graphics. I am not sure whether this is backward compatible or not though. But if you upgrade your docker engine then it is needed otherwise the container is not starting.

**Review by domire8 — APPROVED** [review]

**Pull request merged**